### PR TITLE
Feat [GSW-1676] Launchpad i18n

### PR DESCRIPTION
**Pull request by tfrg**
## Feature
Launchpad page i18n

### CODE REVIEW

**Review by onlyhyde — APPROVED** [review]

File: packages/web/src/components/common/launchpad-modal/launchpad-deposit-modal/LaunchpadDepositModal.tsx [hunk 1]
```diff
@@ -152,28 +178,31 @@ const LaunchpadDepositModal = ({
...
               <li className="list">
-                Double-check to confirm that your deposit amount.
+                {t("Launchpad:modal.deposit.note.list1")}
               </li>
               <li className="list">
-                Only send supported tokens to this deposit address. <br />
-                Depositing any other cryptocurrencies to this <br />
-                launchpad will result in the loss of your funds.
+                <Trans ns="Launchpad" i18nKey={"modal.deposit.note.list2"}>
+                  Only send supported tokens to this deposit address. <br />
```
Comment (jinoosss) [inline packages/web/src/components/common/launchpad-modal/launchpad-deposit-modal/LaunchpadDepositModal.tsx:189]: Please check if it's okay to delete it.

File: packages/web/src/utils/launchpad-get-status-text.ts [hunk 2]
```diff
@@ -26,13 +27,16 @@ export const getStatusText = (
 
   const formatTime = (days: number, hours: number, minutes: number) => {
     if (days > 0) {
-      return `in ${days} days ${hours} hours`;
+      return i18n?.t("Launchpad:common.time.inDaysHours", { days, hours });
```
Comment (jinoosss) [inline packages/web/src/utils/launchpad-get-status-text.ts:30]: In my opinion, it seems better to take over a function that formats i18n as an argument than to use an i18n instance.
It seems like a good idea to have utility functions behave as pure functions.

```javascript
export const getStatusText = (
  type: PROJECT_STATUS_TYPE,
  startTime: string | undefined,
  endTime: string | undefined,
  format: (
    key: string,
    options?: { [key in string]: string | number },
  ) => string,
) => {
    ...
}
```

File: packages/web/src/utils/launchpad-get-tier-number.ts [hunk 3]
```diff
@@ -8,10 +10,23 @@ export const getTierNumber = (tier: TierType): number => {
 export const getTierDuration = (tier: TierType) => {
   switch (tier) {
     case "TIER30":
-      return "1 Month";
+      return i18n?.t("Launchpad:common.tierDuration.1month");
```
Comment (jinoosss) [inline packages/web/src/utils/launchpad-get-tier-number.ts:13]: In my opinion, it seems better to take over a function that formats i18n as an argument than to use an i18n instance.
It seems like a good idea to have utility functions behave as pure functions.

File: packages/web/src/views/launchpad/launchpad-detail/components/launchpad-my-participation/launchpad-my-participation-no-data/LaunchpadMyParticipationNoData.tsx [hunk 4]
```diff
@@ -21,11 +24,23 @@ const LaunchpadMyParticipationNoData = ({
       />
       <div className="banner-text">
         <div className="banner-text-description">
-          Deposit <span>GNS</span>
+          <Trans
+            ns="Launchpad"
+            i18nKey="myParticipation.nodata.text1"
+            components={{ span: <span /> }}
+          >
+            Deposit <span>GNS</span>
```
Comment (jinoosss) [inline packages/web/src/views/launchpad/launchpad-detail/components/launchpad-my-participation/launchpad-my-participation-no-data/LaunchpadMyParticipationNoData.tsx:32]: Please check if it's okay to delete it.

File: packages/web/src/views/launchpad/launchpad-detail/components/launchpad-participate/LaunchpadParticipate.tsx [hunk 5]
```diff
@@ -218,13 +270,13 @@ const LaunchpadParticipate: React.FC<LaunchpadParticipateProps> = ({
         </div>
         <div className="participate-info">
           <div className="participate-info-key">
-            End Date{" "}
+            {t("Launchpad:participate.col.endDate")}{" "}
             <LaunchpadTooltip
               floatingContent={
-                <>
+                <Trans ns="Launchpad" i18nKey="common.tooltip.endDate">
                   The launchpad program you selected <br />
```
Comment (jinoosss) [inline packages/web/src/views/launchpad/launchpad-detail/components/launchpad-participate/LaunchpadParticipate.tsx:277]: Please check if it's okay to delete it.

File: packages/web/src/views/launchpad/launchpad-detail/components/launchpad-participate/LaunchpadParticipate.tsx [hunk 6]
```diff
@@ -258,7 +312,7 @@ const LaunchpadParticipate: React.FC<LaunchpadParticipateProps> = ({
         </div>
       </div>
 
-      {!isLoading && (
+      {/* {!isLoading && (
         <div className="participate-button-wrapper">
           <DepositButton
             isAvailableDeposit={isAvailableDeposit}
```
Comment (jinoosss) [inline packages/web/src/views/launchpad/launchpad-detail/components/launchpad-participate/LaunchpadParticipate.tsx:318]: If it's unused code, it seems like a good idea to delete it.

File: packages/web/src/views/launchpad/launchpad-detail/components/launchpad-project-summary/LaunchpadProjectSummary.tsx [hunk 7]
```diff
@@ -85,13 +88,16 @@ const LaunchpadProjectSummary: React.FC<LaunchpadProjectSummaryProps> = ({
...
-          Tokens Distributed
+          {t("Launchpad:projectSummary.col.tokensDistributed")}
           <LaunchpadTooltip
             floatingContent={
-              <>
+              <Trans
+                ns="Launchpad"
+                i18nKey={"common.tooltip.tokensDistributed"}
+              >
                 The total amount of project tokens <br />
```
Comment (jinoosss) [inline packages/web/src/views/launchpad/launchpad-detail/components/launchpad-project-summary/LaunchpadProjectSummary.tsx:98]: Please check if it's okay to delete it.

**Review by jinoosss — COMMENTED** [review]
LGTM 👍 
I left a few small comments.

**Review by jinoosss — APPROVED** [review]

**Review by jinoosss — APPROVED** [review]

**Review by jinoosss — DISMISSED** [review]

**Review by jinoosss — APPROVED** [review]

**Review by dongwon8247 — APPROVED** [review]

**Pull request merged**